code/game/machinery/cryopod.dm

```diff
@@ -253,7 +253,12 @@
 		/obj/item/key,
 		/obj/item/door_remote,
 		/obj/item/autopsy_scanner,
-		/obj/item/holosign_creator/atmos
+		/obj/item/holosign_creator/atmos,
+		/obj/item/clothing/gloves/color/black/forensics,
+		/obj/item/rcd,
+		/obj/item/rpd,
+		/obj/item/gun/projectile/revolver/detective,
+        /obj/item/clothing/accessory/holster/armpit
```

Comment:
Adding holsters here isn't the right fix. There will be more issues with other items if cryo isn't working properly. It works for guns inside boxes, right? (or does it?)

Fix the code that deletes the contents of holsters. 
Look at `holster.dm:52`, make sure that `holstered.loc` is set properly and that the gun ends up in the holster's `contents`. 
Make sure that the code that makes use of this big list here is properly searching the contents of all objs that get thrown into cryo.

Reply:
Sadly, this is my first time working with DM code, and I have nowhere near an idea of how that kind of stuff works. If anyone would like to help me out, however, I will gladly test the changes and commit them to this PR.